package.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "discordtel",
-  "version": "3.2.0",
+  "name": "dtel",
+  "version": "4.0.0",
   "description": "Discord Telephone",
-  "main": "index.js",
+  "main": "out/index.js",
```

Comment:
Shouldn't this be
```suggestion
  "main": "src/index.ts",
```

Reply:
no, it shouldn't, you don't enter at index.ts

Reply:
do we even compile the code? this isn't a published package after all

but either way there's no index.js and I've never seen out as a folder before (dist instead?)

Reply:
index.ts is not a valid entry point